WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -3876,7 +3876,7 @@ templates.items[1] = {
 }
 
 templates.items[2] = {
-  title = L["On Use Trinkets (Buff)"],
+  title = L["On Use Trinkets (Aura)"],
```

Comment:
Why?

Reply:
figured aura is better term since it also includes debufffs